modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -370,4 +370,68 @@ def query_params(url)
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  context 'mobile flag' do
```

Comment:
When describing a context, start its description with "when" or "with".

```suggestion
  context 'with mobile flag' do
```